web/sites/default/config/views.view.group_services.yml

```diff
@@ -423,7 +423,7 @@ display:
         metatag_display_extender:
           metatags:
             robots: noindex
-      path: group/%group/services
+      path: group/%group/services-old
```

Comment:
Why did you keep this one but delete the news view?

Reply:
View removed